src/main/java/org/veupathdb/service/eda/ds/plugin/correlation/AbstractCorrelationBipartiteNetwork.java

```diff
@@ -33,12 +33,12 @@ public boolean computeGeneratesVars() {
   }
 
   @Override
-  protected void validateVisualizationSpec(CorrelationBipartitenetworkSpec pluginSpec) throws ValidationException {
+  protected void validateVisualizationSpec(CorrelationNetworkSpec pluginSpec) throws ValidationException {
```

Comment:
I don't follow. Why would we be changing to regular networks here? The object creation at the bottom of this file has partition stuff in it that isn't appropriate for unipartite networks